pyOCD/gdbserver/gdbserver.py

```diff
@@ -94,183 +268,146 @@ def setBoard(self, board, stop = True):
         self.flash = board.flash
         self.lock.release()
         return
-        
+
     def run(self):
         self.timeOfLastPacket = time()
```

Comment:
You should be able to remove timeOfLastPacket since packet_io.receive() is blocking